### PR TITLE
*: Add backup Name  into Backup status

### LINKED ISSUE #453: [feature] Provide more details of backup in spec->status of backup CRD
**Is your feature request related to a problem? Please describe.**

stuff for integrate s3 storage client is not planed in the next release of DMP, so we need RadonDB MySQL Operator provide some necessary informations to implement one-step restore function(create a new cluster from a backup).

Existing information:
```
spec:
  clustname: radondb-npbw4y
  historyLimit: 3
  hostname: radondb-npbw4y-mysql-1
  image: radondb/mysql-sidecar:v2.1.2
status:
  completed: true
  conditions:
  - lastTransitionTime: "2022-04-14T07:29:08Z"
    message: ""
    reason: ""
    status: "True"
    type: Complete
```

```restoreFrom``` is missing in Backup CR
```
spec:
  backupSecretName: $S3_SECRET_NAME
  restoreFrom: "backup_20224147294"
```

**Describe the solution you'd like**

Backup detail of PostgreSQL:
```
full backup: 20220415-191402F
            timestamp start/stop: 2022-04-15 19:14:02 +0800 CST / 2022-04-15 19:14:22 +0800 CST
            wal start/stop: 000000010000000000000002 / 000000010000000000000002
            database size: 31.2MiB, backup size: 31.2MiB
            repository size: 3.8MiB, repository backup size: 3.8MiB
            backup reference list:
```

***Our needs***:
- Add cluster name for backup artifact, such as ```radondb-npbw4y_full_backup_20224147294```
- Provide artifact name of backup
- Provide backup type, such as full, incr, diff
- Provide data size

Hope to provide the above information as much as possible

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by acekingke**
### What type of PR is this?

/enhancement
### Which issue(s) this PR fixes?

Fixes #453

### What this PR does?

Summary:
Add BackupName in `backup.Status`

### CODE REVIEW

**Comment by runkecheng** [comment]
add additionalPrinterColumns like: 
```
kubectl get backup   
before:
NAME             AGE
backup-sample1   2s
backup-sample2   2s
after:
NAME             CLUSTER    NAMESPACE   COMPLETED    BACKUP-NAME   SECRET   TIME    
backup-sample1   sample1    default     true         sample_xxx    secret1  yyyy-mm-dd hh:mm:ss
backup-sample2   sample2    default     true         sample_yyy    secret2  yyyy-mm-dd hh:mm:ss
```

File: utils/constants.go [hunk 1]
```diff
@@ -169,6 +169,8 @@ const (
 	PodDisruptionBudget ResourceName = "pdb"
 	// XenonMetaData is the name of the configmap that contains xenon metadata.
 	XenonMetaData ResourceName = "xenon-metadata"
+	// Job Anonations name
```
Comment (andyli029) [inline utils/constants.go:172]: ->Annonations?

**Comment by andyli029** [comment]
@acekingke 
`
Code check / check-crd (pull_request) Failing after 56s — check-crd
`

**Comment by acekingke** [comment]
> @acekingke `Code check / check-crd (pull_request) Failing after 56s — check-crd`

it was fixed.

**Comment by andyli029** [comment]
Ping @Nrehearsal

**Review by Nrehearsal — APPROVED** [review]

**Comment by Nrehearsal** [comment]
[line here](https://github.com/acekingke/radondb-mysql-kubernetes/blob/2f98bfcff561f28915afe6176a211588383d6171/api/v1alpha1/backup_types.go#L32)

@acekingke 

there is a problem, if ```backup.spec.image``` was no set by user, we should set it with ```mysqlcluster.spec.podPolicy.sidecarImage```. use the default value may cause some version compatibility issues

**Review by andyli029 — COMMENTED** [review]
LGTM.

**Pull request merged**